self-core-impl/src/main/java/com/selfxdsd/core/managers/StoredProjectManager.java

```diff
@@ -665,8 +663,7 @@ public void issueLabelsChanged(final Event event) {
             isNotClosed -> new TaskIsRegistered(
                 new IssueEstimationChanged(
                     new UpdateTaskEstimation(
-                        sendReply -> LOG.debug("Send updated estimation"
```

Comment:
@amihaiemil btw, is there a "Log" step? It will be nice to retain this info too.

Reply:
@criske nope, but I'm pretty sure SendReply does some logging internally :D